lib/reporter.ts

```diff
@@ -258,6 +262,9 @@ class ReportPortalReporter extends Reporter {
     this.sessionId = runner.sessionId;
     this.client = client || new ReportPortalClient(this.options.reportPortalClientConfig);
     this.launchId = process.env.RP_LAUNCH_ID;
+    if (this.options.writeLaunchIdToFile) {
+      writeFileSync('./RP_LAUNCH_ID', this.launchId);
```

Comment:
It should be done in service not in reporter https://github.com/BorisOsipov/wdio-reportportal-service/blob/master/src/index.js#L7

Reply:
as well it should be service option

Reply:
I will migrate that over there.

Reply:
Fixed